doc/whats_new/v1.2.rst

```diff
@@ -180,6 +180,26 @@ Changes impacting all modules
   :user:`Olivier Grisel <ogrisel>`, and `Thomas Fan`_,
   :pr:`24556` by :user:`Vincent Maladière <Vincent-Maladiere>`.
 
+- |Enhancement| The `get_feature_names_out` method of the following classes now 
```

Comment:
We need to place this entry in `1.3.rst`.

Reply:
Sure. Working on it.